autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -393,21 +393,29 @@ def fit(train_data,
                 If True, will set Trainer.best_model = Trainer.full_model_dict[Trainer.best_model]
                 This will change the default model that Predictor uses for prediction when model is not specified to the refit_full version of the model that previously exhibited the highest validation score.
                 Only valid if `refit_full` is set.
-            feature_generator_type : :class:`autogluon.utils.tabular.features.auto_ml_feature_generator.AbstractFeatureGenerator` class, default = :class:`autogluon.utils.tabular.features.auto_ml_feature_generator.AutoMLFeatureGenerator`
-                A `FeatureGenerator` class specifying which feature engineering protocol to follow
-                Note: The file containing your `FeatureGenerator` class must be imported into current Python session in order to use a custom class.
-            feature_generator_kwargs : dict, default={}
-                Keyword arguments to pass into the `FeatureGenerator` constructor.
+            feature_generator : `autogluon.utils.tabular.features.generators.abstract.AbstractFeatureGenerator`, default = `autogluon.utils.tabular.features.generators.auto_ml_pipeline.AutoMLPipelineFeatureGenerator()`
+                The feature generator used by AutoGluon to process the input data to the form sent to the models. This often includes automated feature generation and data cleaning.
+                It is generally recommended to keep the default feature generator unless handling an advanced use-case.
                 Valid :class:`autogluon.utils.tabular.features.auto_ml_feature_generator.AutoMLFeatureGenerator` kwargs:
```

Comment:
Docs should clarify how the user is supposed to pass in these kwargs to the feature_generator now.

Reply:
The user passes in an object, not a class, so they would have had to pass in the kwargs when creating the object. Do you think this needs explaining? If so, can you provide the description you would like added?

Reply:
Was minor point, just meant it could be even more explicit like:
 
"To control aspects of the default feature generation process, you can pass in a `AutoMLPipelineFeatureGenerator`  object constructed using some of these kwargs:`

Reply:
Added